client/rpcserver/handlers_test.go

```diff
@@ -814,3 +814,81 @@ func TestHandleLogout(t *testing.T) {
 		}
 	}
 }
+
+func TestHandleOrderBook(t *testing.T) {
+	params := &RawParams{Args: []string{"dex", "42", "0"}}
```

Comment:
A second params set with non-zero nOrders is probably what I'm after with the test coverage for this handler.